packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/creditCardPaymentMethod.gql.js

```diff
@@ -0,0 +1,177 @@
+import gql from 'graphql-tag';
+
+// We disable linting for local fields because there is no way to add them to
+// the fetched schema.
+// https://github.com/apollographql/eslint-plugin-graphql/issues/99
+/* eslint-disable graphql/template-strings */
+
+export const GET_ALL_COUNTRIES = gql`
+    query getAllCountries {
+        countries {
+            available_regions {
+                code
+                id
+                name
+            }
+            id
+        }
+    }
+`;
+
+export const GET_IS_BILLING_ADDRESS_SAME = gql`
+    query getIsBillingAddressSame($cartId: String!) {
+        cart(cart_id: $cartId) {
```

Comment:
In #2322 I'm adding `@connection` directives to all queries and mutations that fetch cart or use a cart ids. You could get ahead of that by doing it in your PR now. I'm not sure if it'll operate entirely correctly if you do it before #2322 gets merged though but once it is merged you definitely should make that change.